### DISCUSSION TITLE
Feature request: Allow changing field override match type after creation

### DISCUSSION THREAD

**Discussion by nicolaerosia** [grafana/grafana#44519] [Dashboards, Panels & Visualizations]
Hello,

It would be great if it would be possible to change the override match type after it was created.
Example:
![image](https://user-images.githubusercontent.com/4127983/142903330-e2ab9fff-4fcf-44aa-ae3c-4195a07373fd.png)

After you create it, you can no longer change the match type:
![image](https://user-images.githubusercontent.com/4127983/142903451-71cef9f5-c6a4-4308-94c5-db99350af7d0.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
